app/components/forms/wizard/other-details-step.js

```diff
@@ -0,0 +1,563 @@
+import Component from '@ember/component';
+import { later } from '@ember/runloop';
+import { observer, computed } from '@ember/object';
+import moment from 'moment';
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6089057)